src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -32,9 +32,9 @@
       <Project>{be8ed8c1-c314-4c4e-a929-64c9c8b3552a}</Project>
       <Name>XunitTraitsDiscoverers</Name>
     </ProjectReference>
-    <ProjectReference Include="..\ProcessTest_ConsoleApp\ProcessTest_ConsoleApp.csproj">
+    <ProjectReference Include="..\ProcessTest_ConsoleApp\System.Diagnostics.Process.TestConsoleApp.csproj">
       <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
-      <Name>ProcessTest_ConsoleApp</Name>
+      <Name>System.Diagnostics.Process.TestConsoleApp</Name>
```

Comment:
I believe this name might be hard coded inside the test itself. Please clean your workspace and check if the test is not complaining that the exe file is missing.